index.html

```diff
@@ -190,66 +218,65 @@ <h2 class="connect-link"><a href="#">Let's connect</a></h2>
           </li>
         </ul>
         <button type="button" class="work-btn">Get my resume</button>
```

Comment:
- [ ] You have already made this as a `button` element so no need to write explicitly with type `button` instead you can give `type= "submit"` or with no type attribute (note that it defaults to a submit type when no type attribute is used). Please make changes to all the places.